.eslintrc.json

```diff
@@ -0,0 +1,43 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "extends": ["airbnb-base", "plugin:react/recommended"],
```

Comment:
- [ ] I am unsure why but `airbnb-base` is causing some compilation issues. Kindly fix this or consider ignoring this file in `.gitignore` as it is not required to run this project.
![Screenshot from 2021-11-01 11-23-57](https://user-images.githubusercontent.com/65476715/139657976-2a846a5a-b69a-411e-abfc-cf98c0adca2e.png)